input/fsh/examples/genomic-study-examples.fsh

```diff
@@ -0,0 +1,213 @@
+* category = $observation-category#laboratory
+* subject = Reference(Patient/genomicPatient)
+* performedDateTime = "2019-03-01"
+* basedOn = Reference(ServiceRequest/genomicServiceRequest)
+* asserter = Reference(Practitioner/practitioner02)
+* reasonCode.coding.system = "http://snomed.info/sct"
+* reasonCode.coding.code = #309529002
+* reasonCode.coding.display = "Lung mass (finding)"
+* note.text = "For technical reasons, PIK3CB was deemed uncallable."
+
+Instance: lungMass-analysis1
+InstanceOf: Procedure
+Usage: #example
+* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomic-study-analysis"
+* extension[0].url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomics-study-analysis-method-type"
+* extension[=].valueCodeableConcept = $sct#117040002 "Nucleic acid sequencing (procedure)"
+* extension[+].url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomics-study-analysis-change-type"
+* extension[=].valueCodeableConcept = $sequenceontology#SO:0001483 "SNV"
+* extension[+].url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomics-study-analysis-change-type"
+* extension[=].valueCodeableConcept = $sequenceontology#SO:0002007 "MNV"
+* extension[+].url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomics-study-analysis-change-type"
+* extension[=].valueCodeableConcept = $sequenceontology#SO:1000032 "delins"
+* extension[+].url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomics-study-analysis-genome-build"
+* extension[=].valueCodeableConcept = $loinc#LA26806-2 "GRCh38"
+* extension[+].url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomics-study-analysis-specimen"
+* extension[=].valueReference = Reference(Specimen/genomicSpecimen)
+* extension[+].url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomics-study-analysis-focus"
+* extension[=].valueReference = Reference(Patient/genomicPatient)
+* extension[+].url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomics-study-analysis-regions-studied"
+* extension[=].valueReference = Reference(DocumentReference/WES-FullSequencedRegion-GRCh38)
+* extension[+].url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomics-study-analysis-regions-called"
+* extension[=].valueReference = Reference(DocumentReference/SimpleVariantAnalysis-called)
+* instantiatesUri = "https://pubmed.ncbi.nlm.nih.gov/33927380/"
+* status = #completed
+* category = $observation-category#laboratory
+* performedDateTime = "2019-03-01T01:01:10-06:00"
+* note.text = "For technical reasons, PIK3CB was deemed uncallable using this method."
+* performer.actor = Reference(Practitioner/practitioner02)
+* performer.function = $CodeSystem-v3-ParticipationType.html#PRF "Performer"
+* report = Reference(DocumentReference/genomicVCFfile-simple)
```

Comment:
@rhdolin - this is also generating a QA error:


Procedure (l1/c2) | error | Procedure.report: max allowed = 0, but found 1 (from http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomic-study-analysis\|2.1.0-SNAPSHOT)
-- | -- | --



this will be true (as I built the profile assuming people would use the extensions for Input/Output and I put 0..0 on 'report' element. We can discuss this, but wanted to mention it.

Reply:
resolved